### DISCUSSION TITLE
WireGuard WebAdmin Now on Docker Hub!

### DISCUSSION THREAD

**Discussion by eduardogsilva** [eduardogsilva/wireguard_webadmin#21] [Announcements]
Hello! 🚀

To enhance our deployment and upgrade process, I've taken a significant step forward by hosting our Docker images on Docker Hub. This update is designed to streamline your experience and make managing your deployments easier.

#### Here's What's New:

- **Instant Access to Docker Images**: No more waiting around for local builds! Our Docker images are now pre-built and stored on Docker Hub, ready for you to pull and use.
- **Simplified Upgrade Path**: With the shift to Docker Hub, updating WireGuard WebAdmin has become simpler. I've updated our upgrade instructions to reflect this change, ensuring you can easily transition to the latest version.
- **Refreshed Deployment Guide**: For anyone setting up WireGuard WebAdmin or planning an upgrade, the README now contains the latest deployment instructions, taking advantage of our Docker Hub images.

If you encounter any issues or have suggestions for enhancement, feel free to [raise an issue](https://github.com/eduardogsilva/wireguard_webadmin/issues) or spark a discussion here. Your insights and experiences are important to us.

**Comment by xgusto** [comment]
I looked for the wireguard_webadmin on the hub docker but couldn't find it.

**Comment by eduardogsilva** [comment]
Hello @xgusto the current url for the main repository is: https://hub.docker.com/r/eduardosilva/wireguard_webadmin

Docker hub is used to host the container images. The main difference is in the install process.

Before dockerhub, the install process would use git clone and build the container images locally.

After I started using docker hub, now you just download the docker compose file and the "docker compose up" process download all container images without having to build each one.

No need to git clone the full repository now. ;)

Cheers!
